modules/auxiliary/scanner/sap/sap_soap_bapi_user_create1.rb

```diff
@@ -81,6 +81,7 @@ def run_host(ip)
         'headers' => {
           'SOAPAction' => 'urn:sap-com:document:sap:rfc:functions',
         },
+        'encode_params' => false,
```

Comment:
i think this is not necessary on the SAP modules because of non payload paramters. I think i looked through the modules and identified the ones which needs the encode_params (by eliminating the fixed and non payload parameters). Or do you want to change all of them?